### ISSUE TITLE
Add option for more AI provider

### ISSUE THREAD

**Issue by suizideFloat** [suitedaces/computer-agent#7]
It would be a very hot and useful feature to be able to add more providers than Anthropic only.
There is a lot of other potent LLMs out there and it might even be possible to use one of them for free, which is a huge benefit when testing with LLMs.

Suggested providers:

OpenAI
Gemini
Mistral
"Custom OpenAI API"

**Comment by RealShocky** [comment]
Everytime i add this it keep's breaking. so as long as you don't mind a little UI change. I can make this work. wont be as fast of feature I've been putting out, but will come soon. if my other commits gets approved. Such as Voice, Prompt Manager, etc.

**Comment by NinjaChunksBackup** [comment]
I would really like to see Gemini to be used for this! The recent model updates of Gemini even made them faster and accurate!

**Comment by RealShocky** [comment]
Okay, you asked for it, so you got it!

Openai provider is now added!
Will add custom provider support next!

**Comment by b00pka** [comment]
I’d love to see Ollama support added as well!
Honestly, I was confused by the word "Local" in the README of this project. I’m not sure if it’s appropriate to call something “local” when it is being controlled by something cloud-based...

**Comment by AliYar-Khan** [comment]
yeah this is something need and also may be use the agents like OpenCode , kilo code under the hood
